WORKSPACE

```diff
@@ -35,6 +35,24 @@ maven_install(
     use_unsafe_shared_cache = True,
 )
 
+maven_install(
+    name = "exclusion_testing",
```

Comment:
I moved this out of `@regression_testing` because other artifacts were bringing animal-sniffer-annotations back in.